models/issue_comment.go

```diff
@@ -1152,9 +1152,7 @@ func DeleteComment(comment *Comment) error {
 }
 
 func deleteComment(e db.Engine, comment *Comment) error {
-	if _, err := e.Delete(&Comment{
-		ID: comment.ID,
-	}); err != nil {
+	if _, err := e.ID(comment.ID).NoAutoCondition().Delete(comment); err != nil {
```

Comment:
what's **NoAutoCondition** for ?

Reply:
Hmm this was included in lunny's change IIRC. I suppose it does disable generating the where clause from the passed comment object whatever is in there, but explicit from the e.ID, but I'm not sure if I fully understand the xorm docs here.

Reply:
```
xorm.io/xorm/engine.go
355:// NoAutoCondition disable auto generate Where condition from bean or not
```

Reply:
Use `NoAutoCondition` will disable to generate conditions from the bean so that we can reuse `comment` object but create a new `comment` object. Then we reduced a memory usage.